inc/saihostif.h

```diff
@@ -359,17 +359,26 @@ typedef enum _sai_hostif_trap_type_t
     SAI_HOSTIF_TRAP_TYPE_BGPV6 = 0x00004004,
 
     /**
-     * @brief BFD traffic (UDP dst port == 3784 or UDP dst port == 4784) to local
-     * router IP address (default packet action is drop)
+     * @brief BFD traffic (UDP dst port == 3784 or UDP dst port == 4784 or
+     * UDP dst port == 6784) to local router IP address (default packet
+     * action is drop)
```

Comment:
Can we add a new trap type like below instead of modifying the existing traps.

Advantages:

* Does not change behavior for existing implementations

* Allows optimization of h/w resources when end user needs only BFD and not micro-BFD.


```
/* 
* @brief BFD traffic (UDP dst port == 6784) to local
* router IP address (default packet action is drop)
*/
SAI_HOSTIF_TRAP_TYPE_BFD_MICRO,
```

Similarly for BFDv6

Reply:
I wish you had brought that up during the meeting today. I feel like the rest of the community should be aware of this request given that they all know we decided on using the same trap type.

In regards to your points:
a) I see it differently, given that someone will get BFD over LAG for free pretty much and it's a common use case rather than not.
b)If someone doesn't need BFD over LAG then they don't need to implement it, correct? An entry with type+port will have to be added to the hardware anyway, it's not easy to call out what is being optimized.

Perhaps we need someone to decide whether we need another meeting for this to a) rediscuss with the community, b) go with your proposal or c) go with the original one.

Reply:
@rhui Could you please share your thoughts on this?

Reply:
It would be good if we can discuss this in the next community meeting. 

In most of the TCAM based Host Tables, we would need one ACL entry per L4 port match.

Reply:
Made changes per your recommendation.